python/graphstorm/eval/evaluator.py

```diff
@@ -707,28 +743,32 @@ def compute_score(self, pred, labels, train=True):
         return scores
 
 class GSgnnRconstructFeatRegScoreEvaluator(GSgnnRegressionEvaluator):
-    """ Evaluator for feature reconstruction using regression scores.
-
-        We treat the prediction results as a 2D float tensor and
-        the label is also a 2D float tensor.
+    """ Evaluator for feature reconstruction tasks using regression scores.
 
-        We compute mse or rmse for it.
+    A built-in evalutor for feature reconstruction tasks. It uses ``mse`` or ``rmse`` as
+    evaluation metrics.
+    
+    This evaluator requires the prediction results to be a 2D float tensor and
+    the label also to be a 2D float tensor, which stores the original features.
```

Comment:
Should "the original features" be changed to "labels"?

Reply:
This is a reconstruction task. The labels are just the original features.